src/Queue/SqsWatchableQueue.php

```diff
@@ -102,7 +102,7 @@ public function count() : int
         ]);
         $total = 0;
         foreach ($attributes as $attributeName) {
-            $total += $result['Attributes'][$attributeName];
+            $total += $result['Attributes'][$attributeName] ?? 0;
```

Comment:
You run into `goaws` missing some attribute right? These are all numbers so `0` is a safe default.